target/snitch_cluster/sw/runtime/common/snitch_cluster_addrmap.h.tpl

```diff
@@ -2,8 +2,17 @@
 // Licensed under the Apache License, Version 2.0, see LICENSE for details.
 // SPDX-License-Identifier: Apache-2.0
 
+
+
 #define CLUSTER_TCDM_BASE_ADDR ${hex(cfg['cluster']['cluster_base_addr'])}
-#define CLUSTER_PERIPH_BASE_ADDR (CLUSTER_TCDM_BASE_ADDR + ${hex(cfg['cluster']['tcdm']['size'] * 1024)})
+#define CLUSTER_TCDM_SIZE ${hex(cfg['cluster']['tcdm']['size'] * 1024)}
```

Comment:
why * 1024?

Reply:
Because they use KByte instead of Byte in CFG file😂